examples/pojo-jar/src/main/java/org/apache/camel/OpenApiRoute.java

```diff
@@ -0,0 +1,53 @@
+// camel-k: name=pojo-jar
+// camel-k: dependency=camel-jackson
+// camel-k: dependency=camel-openapi-java
+// camel-k: dependency=mvn:org.projectlombok:lombok:1.18.22
+// camel-k: resource=file:../../../../../../target/pojo-jar-1.0.0.jar
```

Comment:
Shouldn't it be `// camel-k: resource=file:./target/pojo-jar-1.0.0.jar`?

Reply:
Yes can we avoid all these weird parent folder?

Reply:
I did run the command like this and it worked. I think as it's defined as modeline the cwd/pwd is the java file I define in the kamel command. README => kamel run ./src/main/org/apache/camel/OpenApiRoute.java
So kamel changes the execution location to ./src/main/org/apache/camel/OpenApiRoute.java

Reply:
Ah right, the resources are resolved relative to the current file location.